src/stdlib_linalg.fypp

```diff
@@ -93,6 +94,20 @@ module stdlib_linalg
     #:endfor
   end interface outer_product
 
+  interface kronecker_product
+    !! version: experimental
+    !!
+    !! Computes the Kronecker product of two arrays size M1xN1, M2xN2, returning an (M1*M2)x(N1*N2) array
```

Comment:
```suggestion
    !! Computes the Kronecker product of two arrays of size M1xN1, and of M2xN2, returning an (M1*M2)x(N1*N2) array
```

Reply:
"Also, could you edit the specs and add this implementation, please?"
I am not sure what you mean. Do you mean to commit the suggestion you made? What implementation do you mean I should add? I have an stdlib_linalg_kronecker.fypp file with the implementation in there?

Reply:
Ah shit, I can't read the contributor workflow document apparently.

How should I create the spec? I don't see that in tabulated in the workflow. Should I just make a LaTeX/PDF document?

Reply:
Ah, figured it out. Yeah, I'll work on a spec.

Reply:
I pushed a spec and associated example.